### DISCUSSION TITLE
Logging output of coder_script (stop)

### DISCUSSION THREAD

**Discussion by gjrtimmer** [coder/coder#15908] [General]
I cannot find the output of a `coder_script` resource that has `run_on_stop` activated.
Can somebody point me in the right direction?

**Comment by matifali** [comment]
By default, this gets logged to `/tmp/`, and if it's not persistent we will lose it when the workspace starts next time. 

You can try setting [`log_path`](https://registry.terraform.io/providers/coder/coder/latest/docs/resources/script#log_path-1) to a location that is persistent.
